src/install/demo/appInfo.mdx

```diff
@@ -0,0 +1,38 @@
+---
+componentType: appInfoConfig
+headingText: Use appInfoConfig to show the dropdown options.
+descriptionText: 'Use the appInfoConfig componentType to show the dropdown options. You can also use "recommendedGuided: true" to display guided install or other content below the frontmatter section.'
```

Comment:
Do you think you could bold this text or make it italics or change it's color? My instinct is to make it stylistically stand out: "recommendedGuided: true"

Reply:
Sadly, I can't use Markdown formatting in these frontmatter fields.